PCbuild/python.props

```diff
@@ -32,9 +32,9 @@
 
     <!-- Directories of external projects. tcltk is handled in tcltk.props -->
     <ExternalsDir>$([System.IO.Path]::GetFullPath(`$(PySourcePath)externals\`))</ExternalsDir>
-    <sqlite3Dir>$(ExternalsDir)sqlite-3.8.11.0\</sqlite3Dir>
+    <sqlite3Dir>$(ExternalsDir)sqlite-3.14.2.0\</sqlite3Dir>
```

Comment:
Not sure if this is a change we wanted to actually make...

Reply:
This was simpler than creating a new branch for an old version of sqlite, and @benjaminp basically approved it on IRC (you never replied :)).